src/app/wallets/update-pending-invoices.ts

```diff
@@ -15,9 +15,9 @@ import { NewDealerPriceService } from "@services/dealer-price"
 import { LndService } from "@services/lnd"
 import {
   AccountsRepository,
-  UsersRepository,
```

Comment:
Why the noise? What is the point of switching line 18 and 20 other than that it creates noise in the PR?